hs_core/search_indexes.py

```diff
@@ -100,6 +100,56 @@ def normalize_name(name):
     return normalized.strip()
 
 
+# What extensions indicate each kind of high-level content type
+extension_content_types = {
```

Comment:
I wonder if this should be put in config so we can add/update/delete without code changes

Reply:
Which one? local_settings.py or settings.py? Or something else? Willing to move it, but you need to be more specific.....

Reply:
It belongs in `settings.py` for the default settings.  `local_settings.py` is not in source control and can be used to override the default settings as needed.

Reply:
Done.